src/Mod/PartDesign/Gui/TaskFeatureParameters.cpp

```diff
@@ -164,6 +164,11 @@ bool TaskDlgFeatureParameters::accept()
             // object was already computed, nothing more to do with it...
             Gui::cmdAppDocument(feature, "purgeTouched()");
 
+            // NOTE: @ipatch, below if block should resolve https://github.com/FreeCAD/FreeCAD/issues/24188
```

Comment:
Thanks for the fix. As a general rule we only add comments about closing specific bugs in particularly strange cases, where the code itself looks sketchy so need additional explanation. I'm not sure it's really needed here.
```suggestion
```